src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/QueueItem.cs

```diff
@@ -119,26 +110,30 @@ public async Task StartRequestAsync(TRequestContext? context, CancellationToken
                 _logger.LogWarning($"Could not get request context for {MethodName}");
                 _completionSource.TrySetException(new InvalidOperationException($"Unable to create request context for {MethodName}"));
             }
-            else if (_handler is IRequestHandler<TRequest, TResponse, TRequestContext> requestHandler)
+            else if (MethodHandler is null)
+            {
+                throw new NotImplementedException($"No {nameof(IMethodHandler)} implementation for  {MethodName}.");
```

Comment:
mention taht that means CreateRequestContextAsync wasn't called.

Reply:
Removed now that MethodHandler is no longer nullable